test/jasmine/bundle_tests/plotschema_test.js

```diff
@@ -372,7 +372,6 @@ describe('plot schema', function() {
         var traces = Plotly.PlotSchema.get().traces;
 
         expect(traces.sankey.attributes.hoverinfo.flags.length).toBe(0);
-        expect(traces.choropleth.attributes.showlegend).toBe(undefined, 'no legend attrs for choropleth (for now)');
```

Comment:
We should replace `choropleth` with a trace type that "still" doesn't have support for legend items e.g. `parcoords`

Reply:
... to make sure that this logic

https://github.com/plotly/plotly.js/blob/5ef8a61a0738cf805ba198e766bdd15a72c53c5c/src/plot_api/plot_schema.js#L480-L483

is tested.

Reply:
Used `parcoords` in https://github.com/plotly/plotly.js/pull/4386/commits/a9759a27029ce39dd1071d570b8254d0ea667d00.